cwms-data-api/src/test/java/cwms/cda/api/location/kind/OutletControllerTestIT.java

```diff
@@ -80,6 +97,7 @@ public static void setup() throws Exception {
                 deleteLocation(context, RENAMED_CONDUIT_GATE.getOfficeId(), RENAMED_CONDUIT_GATE.getName());
                 storeLocation(context, NEW_CONDUIT_GATE_2);
                 storeLocation(context, EXISTING_CONDUIT_GATE);
+                storeLocation(context, RATED_OUTLET_LOCATION);
```

Comment:
Add location group deletion just in case.  Use the ID in the NEW_RATED_OUTLET object.

Reply:
This still needs to be resolved

Reply:
Added location group removal